query/src/sql/planner/binder/select.rs

```diff
@@ -31,23 +31,35 @@ use crate::sql::planner::binder::Binder;
 use crate::sql::planner::binder::ColumnBinding;
 use crate::sql::plans::FilterPlan;
 use crate::sql::plans::LogicalGet;
+use crate::sql::plans::Scalar;
 use crate::sql::IndexType;
+use crate::sql::ScalarExprRef;
 use crate::storages::Table;
+use crate::storages::ToReadDataSourcePlan;
+use crate::table_functions::TableFunction;
 
 impl Binder {
     #[async_recursion]
-    pub(super) async fn bind_query(&mut self, query: &Query) -> Result<BindContext> {
+    pub(super) async fn bind_query(
+        &mut self,
+        query: &Query,
```

Comment:
Is it better to move the 'bind_context' as the first function parameter?

Reply:
Got. I'll fix this later.